config/openshift/kustomization.yaml

```diff
@@ -0,0 +1,43 @@
+# Adds namespace to all resources.
+namespace: openshift-kubevirt-ipam-controller-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: kubevirt-ipam-controller-
+
+labels:
+- includeSelectors: true
+  pairs:
+    app: ipam-virt-workloads
+
+resources:
+- ../rbac
+- ../manager
+- ../webhook
+
+patches:
+- path: manager_webhook_patch.yaml
+- path: webhookcainjection_patch.yaml
+- patch: |-
+    - op: add
+      path: /webhooks/0/objectSelector
+      value:
+        matchLabels:
+          kubevirt.io: virt-launcher
+  target:
+    kind: MutatingWebhookConfiguration
+    group: admissionregistration.k8s.io
+    version: v1
+    name: mutating-webhook-configuration
+- patch: |-
+    - op: add
+      path: /metadata/annotations
+      value:
+        service.beta.openshift.io/serving-cert-secret-name: webhook-service
```

Comment:
need to figure out the name of the service here ... the prefix above (line 9) is not playing along here.

maybe consider removing the prefix ?...

Reply:
Ah thanks. I had forgotten about that.

Well, it still makes sense, but, the priority surely drops quite a lot. 

It would still be useful to test drive the feature early on OpenShift though.

Reply:
changing manually only the required bit, the secret need to be "kubevirt-ipam-claims-webhook-service"
maybe you can set it as such for both U/S and D/S
without prefixing stuff (or at least not the secret) and it will work ?

(adapting to kubevirt-ipam-controller-webhook-service once time comes)